src/styles/components/_navbar.scss

```diff
@@ -29,4 +21,44 @@
   @include respond-above('xl') {
     padding: 0 rem(40px);
   }
+
+  &__menu-item,
+  &__authentication-button {
```

Comment:
I don't see any usage for `authentication-button`, do you mean `&__button--account`?